packages/extension-app/src/background/index.ts

```diff
@@ -2,40 +2,106 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+/// <reference types="chrome" />
```

Comment:
Using extensionizer, I don't think we need this anymore

Reply:
ah, turns out I need it for the type signature, e.g in `rpcProxySend(..., port: chrome.runtime.Port)`

without that it'll say `cannot find namespace 'chrome'`

Reply:
wait, but the whole point of using `extensionizer` was that you could write `extension.runtime.port` here, no?

Reply:
For me the point was to access the WebApis for different browsers through a single interface. I don't think it'll help us as far as types though. Probably the better solution would be to do something like

```
declare module 'extensionizer' {
  runtime.Port: chrome.runtime.Port | firefox.runtime.Port | etc....
}
``` ... ?

Reply:
> For me the point was to access the WebApis for different browsers through a single interface

yeah

I initially added these 2 guys:
https://github.com/paritytech/substrate-light-ui/blob/ca904a291334cd61c397e21f8081c585e7d03b29/packages/extension-app/package.json#L22-L23
With these 2, we could use `browser.runtime.Port` everywhere, types work with [this tutorial](https://github.com/kelseasy/web-ext-types#install-it), and the polyfill will replace `browser.*` with `chrome.*` on chrome.

I wanted to see if `extenionizer` worked, but if it doesn't have types we can fallback to these 2, which seem like the standard way to go anyway.